src/engine/llm_engine.cpp

```diff
@@ -14,8 +14,10 @@
 #include "models/model_args.h"
 #include "worker.h"
 
+namespace {
```

Comment:
I don't think this would fix the issue since the counter is globally registered. This change may just convert build error into runtime error. Please check the code in metric.h and check the return type. 

Better way is to use declare instead.